packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -32,5 +32,7 @@ export const messages = {
   hotswap_warn_text:
     "You can't modify Apex classes or triggers during an Apex Debugger session. Save your changes after you're done debugging.",
   created_exception_breakpoint_text: 'Created exception breakpoint for %s.',
-  removed_exception_breakpoint_text: 'Removed exception breakpoint for %s.'
+  removed_exception_breakpoint_text: 'Removed exception breakpoint for %s.',
+  terminate_idle_session_text:
+    'Your debugger session is being terminated because it has been idle for a while.'
```

Comment:
If the user doesn't respond to the prompt, we output this to the Debug Console.

![idleterminated](https://user-images.githubusercontent.com/7286437/32973341-bd851b8c-cbab-11e7-9f11-bc7af69dd149.png)

Reply:
Can we say "it has been idle for an hour" instead of "it has been idle for a while"?